packages/components/src/components/dropdown/dropdown.tsx

```diff
@@ -245,6 +246,7 @@ const Dropdown = ({
     handleBlur,
     has_symbol,
     hint,
+    id,
```

Comment:
@kate-deriv where is it used? if it's not used anywhere, we need to remove it. if it's used implicitly, it's enough to add it to the prop type, here it's not needed :)

Reply:
The `<Dropdown />` was used inside of Duration files and there it was with `id`. As I checked, some styles were applied by `id` and TS was unhappy that `id` is not inside of `<Dropdown />` props.

Reply:
`<Dropdown/>` was used inside of some duration files and there it was written with `id.` I didn't find any style with such `id`, so will delete it from here and in the parent component as I didn't notice any change after removing this `id`.